src/main/webapp/app/guided-tour/tours/course-exercise-overview-tour.ts

```diff
@@ -38,21 +33,21 @@ export const courseExerciseOverviewTour: GuidedTour = {
             orientation: Orientation.TOP,
         }),
         new TextTourStep({
-            highlightSelector: '.row.guided-tour',
+            highlightSelector: '.course-exercise-row.guided-tour',
```

Comment:
NOTE: IMO the guided-tour should add custom classes so that developers know they break nothing by changing class names, class order etc.

This seems out of scope here and probably belongs in  a follow-up